day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,17 @@
+frogs = [*(1..10).reverse_each]
+
+frogs.each do |frog|
+  if frog == 1
+    puts "#{frog} speckled frog sat on a log eating some most delicious bugs."
+    puts "One jumped in the pool where its nice and cool,"
+    puts "then there were no more speckled frogs!"
+  elsif frog == 2
+    puts "#{frog} speckled frogs sat on  a log eating some most delicious bugs."
+    puts "One jumped in the pool where its nice and cool,"
+    puts "then there was 1 speckled frogs."
+  else
+    puts "#{frog} speckled frogs sat on  a log eating some most delicious bugs."
+    puts "One jumped in the pool where its nice and cool,"
+    puts "then there were #{frog - 1} speckled frogs."
+  end
+end
```

Comment:
Nice use of iteration and attention to detail with how phrases change. I'd challenge you to think of how to eliminate as much repetition as possible - especially if it's only the last line changing in each conditional.